engine/src/main/java/org/destinationsol/game/screens/MainScreen.java

```diff
@@ -144,7 +145,9 @@ public MainScreen(float resolutionRatio, RightPaneLayout rightPaneLayout, Contex
         cameraControl = new CameraKeyboardControl(gameOptions, controls);
 
         consoleControl = new SolUiControl(null, true, Input.Keys.GRAVE);
+        consoleControl2 = new SolUiControl(null, true, Input.Keys.F1);
```

Comment:
Can we call this `consoleControlF1` instead? And the other `consoleControlGrave`. Or something similar so we know what's going on down the line